docs/workforce-management-framework/topics/pro-using-demo-app.adoc

```diff
@@ -0,0 +1,103 @@
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as Workorders. Workers can view their Workorder via a mobile applicaion and administrators
+can manage workorders via the web portal.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.
+
+image::/shared/images/portal.png[portal]
+
+=== Create Workflow
+
+Click on Workflow in the left panel. To create a workflow click on the `CREATE WORKFLOW` button. This will bring you to the create workflow screen where you can add a title.
+Click `CREATE WORKFLOW` button to continue.
+
+image::/shared/images/create-workflow.png[create workflow]
+
+The add step screen will render and will allow you to name your step and select a one of the example step types from a dropdown (Developed custom steps will appear in the dropdown). 
+Click `ADD STEP` to submit your new step as part of your Workflow. 
+
+image::/shared/images/workflow-nextstep.png[next step workflow]
+
+You have options to delete, update workflows in the top right hand side with the trash and pencil icons. You also can use the circle with the plus icon to create new workflows. This new Workflow 
+can be used to create or update workorders from the workorders screen as shown in the image below. 
+
+image::/shared/images/new-workflow.png[select new workflow]
+
+The new workflow will appear in the list in the center panel after a refresh.
+You can access existing you can access existing workflows by clicking on them in the list in the center panel. Options to edit the workflow are the same 
+as the create.
+
+The portal is the `Workflow` functionality this allows you to create or modify a workflow.
+Workflows are made up of steps, there are a couple of example steps implemented in the demo application. Click Workflow on the 
+left panel to access the the Workflow screens .
+
+image::/shared/images/workflows.png[workflows]
+
+=== Create Workorder
+
+Workorders are individual instances of a Workflow that can be assigned to a worker.
+Click on Workorder in the left panel. Create new workorders by clicking the `NEW WORKORDER` button on the right panel. Once clicked
+it will render a form for creating a new Workorder. See example below
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new Workorder will appear in the list on middle panel after a refresh. 
```

Comment:
Add info: Workorder is created with the clone of the existing workflow. Future edits of the workflow will not affect current workorder. If you want to change current workorder flow you will need to remove it and create a new one.